ledger_secure_sdk_sys/build.rs

```diff
@@ -71,48 +71,212 @@ const CCID_FILES: [&str; 9] = [
     "lib_stusb/STM32_USB_Device_Library/Class/CCID/src/usbd_ccid_if.c",
 ];
 
-#[derive(Debug)]
+#[derive(Debug, PartialEq)]
 enum Device {
     NanoS,
     NanoSPlus,
     NanoX,
 }
 
+impl Device {
+    fn to_string(&self) -> String {
+        match self {
+            Device::NanoS => "nanos",
+            Device::NanoSPlus => "nanos2",
+            Device::NanoX => "nanox",
+        }
+        .to_string()
+    }
+}
+
+struct SDKInfo {
+    pub bolos_sdk: PathBuf,
+    pub api_level: u32,
+    pub target_id: String,
+    pub target_name: String,
+    pub c_sdk_name: String,
+    pub c_sdk_hash: String,
+    pub c_sdk_version: String,
+}
+
+impl SDKInfo {
+    pub fn new() -> Self {
+        SDKInfo {
```

Comment:
you can add a `#[derive(Default)]` above `struct SDKInfo` line 92, then simply have
```
pub fn new() -> Self {
    SDKInfo::default()
}
```

Reply:
Done.